docs/how to/use-transferbench.rst

```diff
@@ -1,32 +1,35 @@
 .. meta::
-  :description: TransferBench documentation 
+  :description: TransferBench documentation
   :keywords: TransferBench, API, ROCm, documentation, HIP
 
+.. _using-transferbench:
 
+---------------------
 Using TransferBench
 ---------------------
-  
-Users have control over the SRC and DST memory locations by indicating memory type followed by the device index. TransferBench supports the following:
 
-* coarse-grained pinned host memory
-* unpinned host memory
-* fine-grained host memory
-* coarse-grained global device memory
-* fine-grained global device memory
-* null memory (for an empty transfer).
+You can control the SRC and DST memory locations by indicating the memory type followed by the device index. TransferBench supports the following memories:
+
+* Coarse-grained pinned host memory
+* Unpinned host memory
+* Fine-grained host memory
+* Coarse-grained global device memory
+* Fine-grained global device memory
+* Null memory (for an empty transfer)
 
-In addition, users can determine the size of the transfer (number of bytes to copy) for their tests.
+In addition, you can determine the size of the transfer (number of bytes to copy) for the tests.
 
-Users can also specify executors of the transfer. The options are CPU, kernel-based GPU, and SDMA-based GPU (DMA) executors. TransferBench also provides the option to choose the number of sub-executors. In case of a CPU executor this argument specifies the number of CPU threads, while for a GPU executor it defines the number of compute units (CU). If DMA is specified as the executor, the sub-executor argument determines the number of streams to be used.
+You can also specify executors of the transfer. The options are CPU, kernel-based GPU, and SDMA-based GPU (DMA) executors. TransferBench also provides the option to choose the number of Sub-Executors (SE). The number of SEs specify the number of CPU threads in case of a CPU executor and the number of compute units (CU) for a GPU executor.
+For a DMA executor, the SE argument determines the number of streams to be used.
 
-Refer to the following example to use TransferBench.
+You can specify the transfers in a configuration file or use preset configurations for transfers.
 
---------------------
-ConfigFile format
---------------------
+Specifying transfers in a configuration file
+----------------------------------------------
 
-A Transfer is defined as a single operation where an executor reads and adds together values from Source (SRC) memory locations, then writes the sum to destination (DST) memory locations.
-This simplifies to a simple copy operation when dealing with single SRC/DST.
+A transfer is defined as a single operation where an executor reads and adds together values from SRC memory locations, then writes the sum to the DST memory locations.
```

Comment:
This sentence is awkward. The "then" is messed up. Should this be two sentences or reworded.

Reply:
Done. "A transfer is defined as a single operation where an executor reads and adds together values from SRC memory locations followed by writing the sum to the DST memory locations."